### PR TITLE
there will be no showing of next alarm time until the user create a alarm time 

### PR DESCRIPTION
**Pull request by nikkivirtuoso**
### Description

I have done some minor changes and along with this i have resolved nits that make the code seems clear and easy to understand

### Proposed Changes

There are some nits changes and some changes in the alarm_ring_controller.dart

## Fixes #(284)
the issue was about showing the next alarm time while there was not any alarm set by the user, now it will not show until the user create an alarm 

## Screenshots

Screenshot 2024-01-15 at 10 10 41 PM

## Checklist

- [ ] Tests have been added or updated to cover the changes
- [ ] Documentation has been updated to reflect the changes
- [ ] Code follows the established coding style guidelines
- [ ] All tests are passing

### CODE REVIEW

**Comment by nikkivirtuoso** [comment]
@MarkisDev sir please have a look

**Comment by cfsmp3** [comment]
Please edit the subject so it says what it actually does :-)

**Comment by nikkivirtuoso** [comment]
@cfsmp3 sir, please have a look i have updated the subject

File: lib/app/modules/settings/views/language_menu.dart [hunk 1]
```diff
@@ -35,30 +34,41 @@ class _LanguageMenuState extends State<LanguageMenu> {
...
+                      child: Text(
+                        "${e.value['description']}",
+                        style: const TextStyle(
+                            fontSize: 16, fontWeight: FontWeight.bold),
+                      ),
+                    );
+                  }).toList(),
+                  onChanged: (newValue) {
+                    widget.controller.updateLocale(newValue!);
+                    setState(() {
```
Comment (MarkisDev) [inline lib/app/modules/settings/views/language_menu.dart:57]: We're using GetX for state management so you'll have to fix that.

**Review by MarkisDev — CHANGES_REQUESTED** [review]

**Comment by MarkisDev** [comment]
Also, I couldn't find the fix for #284 in this PR. Am I missing something? Please let me know :)

**Comment by MarkisDev** [comment]
Closing since its stale, please feel free to make a new one if there's anything else.

**Pull request closed without merging**